lib/bundler/cli/doctor.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require "rbconfig"
+
+module Bundler
+  class CLI::Doctor
+    DARWIN_REGEX = /\s+(.+) \(compatibility /
+    LDD_REGEX = /\t\S+ => (\S+) \(\S+\)/
+
+    attr_reader :options
+
+    def initialize(options)
+      @options = options
+    end
+
+    def otool_available?
+      system("otool --version 2>&1 >/dev/null")
```

Comment:
we can't use `/dev/null` and need to use `Bundler::NULL` instead because of Windows